Env Var capture updates (#3063)
elastic/security-docs: docs/cloud-native-security/environment-variable-capture.asciidoc

To set up environment variable capture for an {agent} policy:

. Go to **Security -> Manage -> Policies**.
. Select an {agent} policy.
. Click *Show advanced settings*.
. Scroll down or search for `linux.advanced.capture_env_vars`.
. Enter the names of env vars you want to capture, separated by commas. For example: `PATH,USER`
. Click *Save*.

[role="screenshot"]
image::images/env-var-capture.png[The "linux.advanced.capture_env_vars" advanced agent policy setting]

> [discrete]

Reviewer: ```suggestion
[[find-cap-env-vars]]
[discrete]
```

== Find captured environment variables
Captured environment variables are associated with process events, and appear in each event's `process.env_vars` field.

> This means that if you want to view environment variables in, for example, the *Events* table on the *Hosts* page, you can:

Reviewer: ```suggestion
To view environment variables in the *Events* table:
```

> . Navigate to the table and click *Fields*.

Reviewer: ```suggestion
. Click the *Event* tab on the *Hosts*, *Network*, or *User* pages, then click *Fields* in the Events table.
```

> . Search for `process.env_vars`, select it, and click *Close*.

Reviewer: ```suggestion
. Search for the `process.env_vars` field, select it, and click *Close*.
```

To set up environment variable capture for an {agent} policy:

. Go to **Security -> Manage -> Policies**.
. Select an {agent} policy.
. Click *Show advanced settings*.
. Scroll down or search for `linux.advanced.capture_env_vars`.
. Enter the names of env vars you want to capture, separated by commas. For example: `PATH,USER`
. Click *Save*.

[role="screenshot"]
image::images/env-var-capture.png[The "linux.advanced.capture_env_vars" advanced agent policy setting]

[[find-cap-env-vars]]
[discrete]
== Find captured environment variables
Captured environment variables are associated with process events, and appear in each event's `process.env_vars` field.

To view environment variables in the *Events* table:

> . Click the *Event* tab on the *Hosts*, *Network*, or *User* pages, then click *Fields* in the Events table.

Author: ```suggestion
. Click the *Event* tab on the *Hosts*, *Network*, or *Users* pages (*Security -> Explore*), then click *Fields* in the Events table.
```

> . Click the *Event* tab on the *Hosts*, *Network*, or *Users* pages (*Security -> Explore*), then click *Fields* in the Events table.

Author: ```suggestion
. Click the *Events* tab on the *Hosts*, *Network*, or *Users* pages (*Security -> Explore*), then click *Fields* in the Events table.
```
